py/frequenz/client/electricity_trading/_types.py

```diff
@@ -879,22 +879,28 @@ def from_pb(cls, order: electricity_trading_pb2.Order) -> Self:
         Returns:
             Order object corresponding to the protobuf message.
         """
+        print(order.stop_price.amount)
+        print(type(order.stop_price.amount))
+        if not order.stop_price.amount.value:
+            print("stop price")
```

Comment:
I'm curious. Is it ok to have print statements in the code?

If they are there for debug purposes we can have them behind a flag

Reply:
no it's not. I just forgot to remove them.

Reply:
fixed!

Reply:
Cheers! (sorry to but in)

Reply:
It's appreciated ;).